### ISSUE TITLE
molecule converge command failes

### ISSUE THREAD

**Issue by eliba** [nierdz/ansible-role-nextcloud#50]
Hi, nice role!

I tried to run `molecule converge` on this role, but it fails at the following step:
```
TASK [geerlingguy.nginx : Remove legacy vhosts.conf file.] *********************
ok: [instance]

TASK [geerlingguy.nginx : Copy nginx configuration in place.] ******************
changed: [instance]

TASK [geerlingguy.nginx : Ensure nginx service is running as configured.] ******
fatal: [instance]: FAILED! => {"changed": false, "msg": "Unable to start service nginx: Job for nginx.service failed because the control process exited with error code.\nSee \"systemctl status nginx.service\" and \"journalctl -xe\" for details.\n"}

RUNNING HANDLER [geerlingguy.nginx : restart nginx] ****************************

RUNNING HANDLER [geerlingguy.nginx : reload nginx] *****************************

PLAY RECAP *********************************************************************
instance                   : ok=15   changed=9    unreachable=0    failed=1    skipped=8    rescued=0    ignored=0

CRITICAL Ansible return code was 2, command was: ansible-playbook --inventory /home/ebackmund/.cache/molecule/ansible-role-nextcloud/default/inventory --skip-tags molecule-notest,notest /home/ebackmund/repos/ansible-role-nextcloud/molecule/default/prepare.yml
```

When inspecting the container via `molecule login` and running `journalctl -u nginx`:
```
Jun 22 08:23:16 instance systemd[1]: Starting nginx - high performance web server...
Jun 22 08:23:16 instance nginx[5990]: nginx: [emerg] cannot load certificate "/etc/nginx/nextcloud.local.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/nextcloud.local.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
Jun 22 08:23:16 instance systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Jun 22 08:23:16 instance systemd[1]: nginx.service: Failed with result 'exit-code'.
Jun 22 08:23:16 instance systemd[1]: Failed to start nginx - high performance web server.
Jun 22 08:23:16 instance systemd[1]: nginx.service: Consumed 4ms CPU time.
```
Where do you get the necessary SSL certificates from? In my personal playbook, I added a role to get the certificates, but what about a molecule run, did I miss something?

**Comment by eliba** [comment]
Found it in `molecule/defaults/prepare.yml`. Made molecule create a fresh SSL keypair in case no existing one was found. Please see #51.

**Comment by nierdz** [comment]
Let's continue this discussion inside #51

**Comment by nierdz** [comment]
Resolved in #56
